### DISCUSSION TITLE
DAI 2023-2024 - Practical work 1 - Olin Bourquin and Kevin Farine

### DISCUSSION THREAD

**Discussion by OlinB** [heig-vd-dai-course/heig-vd-dai-course#62] [Show and tell]
.

**Comment by KevinFarine** [comment]
Hey, it's me !

**Comment by hadrylouis** [comment]
Could you please tell us a bit about what you plan to do for this practical work? :)

**Reply by KevinFarine** [reply]
Sure !
We plan to create a CLI who searches for all occurences of a specific word and replaces them with another specific word.

**Reply by ludelafo** [reply]
Perfect! As discussed yesterday, you can even add more features to the CLI to set the following:

- Only the whole words should be substituted
- Substrings are substituted as well
-  How to handle the performances of the substitution
- ...

Feel free to add more to the CLI :) But, as always, start small and build something bigger after you managed to create something easy to use and understand.

**Comment by OlinB** [comment]
[repo](https://github.com/OlinB/heig-vd-dai-pw-1)
last commit hash: `30d761c`
